plugins/modules/source_control/gitlab/gitlab_service.py

```diff
@@ -0,0 +1,485 @@
+                     "note_channel": {"type": "str"}, "tag_push_channel": {"type": "str"}, "pipeline_channel": {"type": "str"},
+                     "wiki_page_channel": {"type": "str"}, "push_events": {"type": "bool"}, "issues_events": {"type": "bool"},
+                     "confidential_issues_events": {"type": "bool"}, "merge_requests_events": {"type": "bool"}, "note_events": {"type": "bool"},
+                     "confidential_note_events": {"type": "bool"}, "tag_push_events": {"type": "bool"}, "pipeline_events": {"type": "bool"},
+                     "wiki_page_events": {"type": "bool"}},
+           "slack-slash-commands": {"token": {"required": 1, "type": "str", "no_log": True}},
+           "teamcity": {"teamcity_url": {"required": 1, "type": "str"}, "build_type": {"required": 1, "type": "str"},
+                        "username": {"required": 1, "type": "str"}, "password": {"required": 1, "type": "str", "no_log": True}},
+           "unify-circuit": {"webhook": {"required": 1, "type": "str", "no_log": True}, "push_events": {"type": "bool"},
+                             "issues_events": {"type": "bool"}, "confidential_issues_events": {"type": "bool"}, "merge_requests_events": {"type": "bool"},
+                             "note_events": {"type": "bool"}, "confidential_note_events": {"type": "bool"}, "tag_push_events": {"type": "bool"},
+                             "pipeline_events": {"type": "bool"}, "wiki_page_events": {"type": "bool"}},
+           "webex-teams": {"webhook": {"required": 1, "type": "str", "no_log": True}, "push_events": {"type": "bool"},
+                           "issues_events": {"type": "bool"}, "confidential_issues_events": {"type": "bool"}, "merge_requests_events": {"type": "bool"},
+                           "note_events": {"type": "bool"}, "confidential_note_events": {"type": "bool"}, "tag_push_events": {"type": "bool"},
+                           "pipeline_events": {"type": "bool"}, "wiki_page_events": {"type": "bool"}},
+           "youtrack": {"project_url": {"required": 1, "type": "str"}, "issues_url": {"required": 1, "type": "str"}}}
+
+
+def init_definitions():
+    for params in SRV_DEF.values():
+        for name, definition in params.items():
+            if 'type' in definition:
+                definition['type'] = bool if definition['type'] == 'bool' else \
+                    int if definition['type'] == 'int' else \
+                    str if definition['type'] == 'str' else \
+                    definition['type']
+            if name in GitLab_Services.credentialParams:
+                definition['no_log'] = True
+    return SRV_DEF
+
+
+class GitLab_Services(object):
+    HOOK_EVENTS = ['push', 'issues', 'confidential_issues', 'merge_requests', 'tag_push', 'note', 'confidential_note', 'job', 'pipeline', 'wiki_page']
+    credentialParams = ['password', 'token', 'api_key']
```

Comment:
Shouldn't this be
```suggestion
    CREDENTIAL_PARAMS = ['password', 'token', 'api_key']
```
then?